SwiftyJSON.xcodeproj/project.pbxproj

```diff
@@ -670,7 +684,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 7.0;
```

Comment:
iOS 8 seems to be the minimum deployment target supported by Xcode 8 / Swift 2.3